src/System.Collections.Concurrent/tests/ProducerConsumerCollectionTests.cs

```diff
@@ -731,25 +731,19 @@ public void ManyConcurrentAddsTakes_EnsureTrackedCountsMatchResultingCollection(
         }
 
         [Theory]
-        [InlineData(ConcurrencyTestSeconds)]
-        public void ManyConcurrentAddsTakes_CollectionRemainsConsistent(double seconds)
+        [InlineData(1000)]
```

Comment:
>(Or the test needs to be reworked entirely to allow it to run for a specific amount of time without concern for growing the collection too large.)

My worry with the timing loop is that it's so susceptible to test interference i.e. other concurrent tests hogging the machine. There's no minimum or maximum number of items, so we can either have a useless test if we don't do enough operations or an OOM if we do too many. I had originally planned on just adding a max collection size, but that doesn't resolve the interference issue.